packages/forms-web-app/src/lib/session.js

```diff
@@ -1,22 +1,55 @@
 const config = require('../config');
+const logger = require('./logger');
+const { createClient } = require('redis');
+const ConnectRedis = require('connect-redis');
 
-module.exports = () => {
-	const { sessionSecret } = config.server;
+const configureSessionStore = (session) => {
+	const RedisStore = ConnectRedis(session);
 
-	if (!sessionSecret) {
-		throw new Error('Session secret must be set');
-	}
+	if (!config.server.sessionSecret) throw new Error('Session secret must be set');
 
-	const sessionConfig = {
-		secret: sessionSecret,
+	let sessionConfig = {
+		secret: config.server.sessionSecret,
 		resave: false,
 		saveUninitialized: true,
-		cookie: {}
+		cookie: {
+			maxAge: 1000 * 60 * 60 * config.server.sessionLengthInHours
+		}
 	};
 
 	if (config.server.useSecureSessionCookie) {
-		sessionConfig.cookie.secure = true; // serve secure cookies
+		sessionConfig.cookie.secure = true;
+	}
+
+	if (config.featureFlag.useRedisSessionStore) {
```

Comment:
Could this a be a func something like configureRedis ? (Just do it inside the file and leave the tests test)

It should help separate out Redis and the default session store. 

Something like - 
`if (config.featureFlag.useRedisSessionStore) sessionConfig.store = redisThing(session)`

Connect redis  `const RedisStore = ConnectRedis(session);` could be put inside the function aswell

Reply:
Yeah that sounds good to me 👍